### PR TITLE
New MetaManager and ContentVariables cleanup

### PR DESCRIPTION
**Pull request by jsimck**

### CODE REVIEW

File: packages/core/src/page/handler/PageMetaHandler.ts [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+      for (const [attrName, attrValue] of Object.entries(attributes)) {
+        const sanitizedAttrValue = this.#sanitizeValue(attrValue);
+
+        // Skip empty values
+        if (sanitizedAttrValue === null) {
+          continue;
+        }
+
+        metaTag?.setAttribute(attrName, sanitizedAttrValue);
```
Comment (mjancarik) [inline packages/core/src/page/handler/PageMetaHandler.ts:101]: Why do you use `metaTag?.setAttribute`?

File: packages/core/src/page/handler/PageMetaHandler.ts [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+      }
+
+      document?.head.appendChild(metaTag);
```
Comment (mjancarik) [inline packages/core/src/page/handler/PageMetaHandler.ts:104]: I would write `document?.head?.appendChild(metaTag);` for missing head.

File: packages/core/src/page/handler/PageMetaHandler.ts (hunk 1 above)
Comment (jsimck) [reply packages/core/src/page/handler/PageMetaHandler.ts]: Typo, fixed

File: packages/core/src/page/handler/PageMetaHandler.ts (hunk 2 above)
Comment (jsimck) [reply packages/core/src/page/handler/PageMetaHandler.ts]: Well, the post handlers run pretty late so I don't thing there will be an issue but I've fixed it :)

Comment (mjancarik) [reply packages/core/src/page/handler/PageMetaHandler.ts]: for misssing head in html :-)

**Pull request merged**